packages/ui-components/src/components/ReferenceTooltip.tsx

```diff
@@ -1,36 +1,36 @@
 /*
  * Tupaia
- * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
- *
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
+
 import React from 'react';
-import { Typography, Tooltip, withStyles } from '@material-ui/core';
+import { Tooltip, Typography, withStyles } from '@material-ui/core';
 import InfoRoundedIcon from '@material-ui/icons/InfoRounded';
 import styled from 'styled-components';
 
-const DEFAULT = 'default';
-const TILE_SET = 'tileSet';
-const MAP_OVERLAY = 'mayOverlay';
+type ButtonType = 'default' | 'tileSet' | 'mapOverlay';
 
 type ButtonProps = {
-  $buttonType: string;
+  $buttonType: ButtonType;
 };
 
 const IconButton = styled(InfoRoundedIcon)<ButtonProps>`
-  font-size: ${({ $buttonType }) => ($buttonType === MAP_OVERLAY ? '20px' : '16px')};
-  margin-top: ${({ $buttonType }) => ($buttonType === MAP_OVERLAY ? '3px' : '0px')};
-  margin-bottom: ${({ $buttonType }) => ($buttonType === TILE_SET ? '-1px' : '0px')};
-  color: grey;
+  color: rgba(255, 255, 255, 70%);
```

Comment:
Oh, this isn’t obvious at all, but when the ℹ️ icon appears over the blue here, the contrast and hue are all wrong. The ✌️real✌️ solution here is to define a more comprehensive colour palette in `theme.ts`, but I was just being a bit lazy here since it was already a hard-coded colour 😅

![Screenshot 2024-01-23T151318](https://github.com/beyondessential/tupaia/assets/33956381/10e1bf06-c201-4770-9cff-499526c4c7b7)